platform/core/src/measurements/tools/length.js

```diff
@@ -1,7 +1,7 @@
 const displayFunction = data => {
   let lengthValue = '';
   if (data.length) {
-    lengthValue = data.length.toFixed(2) + ' mm';
+    lengthValue = Number(data.length).toFixed(2) + ' mm';
```

Comment:
Unfortunately, the number conversion here isn't sufficient since it breaks in other places in the cornerstone-tools if it is left as a string. The reason this is here seems to be because the dcmjs parser is returning back a string for what should be a javascript/json number. Decimal Strings are most logically translated as numbers. The DICOM JSON data type mapping does just this. http://dicom.nema.org/medical/dicom/current/output/html/part18.html#table_F.2.3-1

Here is where the string value is parsed out of the DICOM in the dcmjs library: https://github.com/dcmjs-org/dcmjs/blob/2037c76b65b618e1fde57a50eb8fc94a99126dd1/src/ValueRepresentation.js#L436

We could work around in a slightly better way by converting all of the erroneous decimal strings to numbers when the data is read from the SR before it becomes tool state. Do you think that the dcmjs library might fix this problem since it is producing bad tool state data?

Reply:
I found a GH issue for this in the dcmjs project: https://github.com/dcmjs-org/dcmjs/issues/53

Reply:
Yeah, good catch @cmcgee-mac, 
Steve just opened a PR in order to fix it on dcmjs
https://github.com/dcmjs-org/dcmjs/pull/79